simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -2279,6 +2240,26 @@
         "supportsReservations": true,
         "supportsSimplyESync": true
     },
+    {
+        "authPasscodeLength": 0,
+        "id_numeric": 1263,
+        "id_uuid": "urn:uuid:5278562c-d642-4fda-ad7e-1613077cfb8d",
```

Comment:
I don't see `id_uuid` used in the code either.

Reply:
That's a little worrying to me; I'm assuming you use the `id_numeric` instead?

The `id_uuid` is a replacement for `id_numeric`. These UUIDs are how the library registry identifies the libraries. I thought we switched over already but maybe it was only on Android.

If we haven't switched yet, there'll be a migration step for people who have data on their phones that's filed under the numeric IDs, and you'll need to keep the id_numeric -> id_uuid mapping around for quite a while, even when the rest of Accounts.json is gone, until we're sure that everyone is migrated.

Reply:
Yes, `id_numeric` is used instead, and you're right, we'll need that for a long time, since it's used to store user logins.